ctx.go

```diff
@@ -848,9 +848,9 @@ func (c *Ctx) Path(override ...string) string {
 	return c.path
 }
 
-// Protocol contains the request protocol string: http or https for TLS requests.
+// Scheme contains the request scheme: http or https for TLS requests.
 // Use Config.EnableTrustedProxyCheck to prevent header spoofing, in case when your app is behind the proxy.
-func (c *Ctx) Protocol() string {
+func (c *Ctx) Scheme() string {
```

Comment:
This is breaking change i think.

Reply:
Yes, it is. I raised this in the discussion as well. The tests have all passed though.